### DISCUSSION TITLE
Issue With Version Updated By You

### DISCUSSION THREAD

**Discussion by sanju-cmd** [prisma/prisma#14049] [Q&A] [answered]
Error: @prisma/client did not initialize yet. Please run "prisma generate" and try to import it again.
In case this error is unexpected for you, please report it in https://github.com/prisma/prisma/issues    ????

**Comment by austincrim** [comment, marked as answer]
Hey @sanju-cmd 👋,

Is this issue occurring after updating the Prisma Client? What version did you update to and which version did you update from?

Have you tried running `npx prisma generate`?

**Comment by sanju-cmd** [comment]
Hi sir , issue fixed with npx install prisma client and then npx prisma
generate .  Thank you sir for replying.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
